2.4/root/usr/share/container-scripts/mongodb/mongodb.conf.template

```diff
@@ -1,7 +1,6 @@
 # mongodb.conf
 
-port = 27017
-pidfilepath = ${MONGODB_PID_FILE}
```

Comment:
Same as in https://github.com/openshift/mongodb/pull/147#discussion_r61854214 MongoDB scripts won't use this variable. 

@rhcarvalho Who can use this? Is it somehow possible use this file with PID in other dependent applications in different containers?

What docs please? I can't find any...